kaggle_environments/envs/halite/helpers.py

```diff
@@ -16,138 +16,12 @@
 from enum import Enum, auto
 from functools import wraps
 from typing import *
```

Comment:
I could be wrong, but you might not need this now with the classes moved up to the central `helpers.py` now?

Reply:
Oh cool, I didn't know that some types are implicitly imported.

Reply:
Nevermind, looks like this is still needed for all other types in the file.